### ISSUE TITLE
Why is ServiceFacilitySet, SiteFacilitySet not a VersionedChild?

### ISSUE THREAD

**Issue by skinkie** [NeTEx-CEN/NeTEx#406]
https://github.com/NeTEx-CEN/NeTEx/blob/master/xsd/netex_framework/netex_reusableComponents/netex_facility_version.xsd#L306

**Comment by skinkie** [comment]
You want to investigate:
'LineSection', 'ServiceAccessRight2', 'ActivatedEquipment', 'SimpleValidityCondition', 'TravelSpecification2', 'FareSection', 'GroupOfStopPlaces', 'ServiceFacilitySet', 'TimetabledPassingTimeView', 'EstimatedPassingTimeView', 'ComplexFeature', 'GeneralSection', 'SiteFacilitySet', 'OtherPlaceEquipment', 'CommonSection', 'TargetPassingTimeView', 'FareContractEntry', 'ObservedPassingTimeView', 'ServiceAccessRight1', 'SimpleFeature', 'PassengerCapacity'

These are not part of GeneralFrame. My guess is they are invalidly not VersionedChild.

**Comment by Aurige** [comment]
I think that it makes sense to complement them with a `substitutionGroup="VersionedChild"`

**Comment by skinkie** [comment]
@Aurige PassengerCapacity is also a really strange variant. It is an EntityInVersion with an optional id attribute... why make such exceptions?

**Comment by Aurige** [comment]
I can't see this:
![image](https://user-images.githubusercontent.com/3328999/211750982-d9b9f124-b771-4240-885b-725a37609673.png)

**Comment by skinkie** [comment]
@Aurige you can here:
https://github.com/NeTEx-CEN/NeTEx/blob/master/xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd#L613

**Comment by Aurige** [comment]
I have this, but this is not an additional Id, it is the specialisation of the Id to a  type="PassengerCapacityIdType" (the original Id being inherited from ENTITY IN VERSION ...)

**Comment by skinkie** [comment]
Yes, but it now makes an entity in version with an _optional_ id.

**Comment by Aurige** [comment]
Id is not always necessary when something is mainly embedded in the xml hierarchy, and not a standalone object... but ok, that's not best for database... also in such case there is one single use of the info (no reuse) so you can define the id you want

**Comment by skinkie** [comment]
I don't think we should promote inconsistent behavior.

**Comment by nick-knowles** [comment]
FacilitySets  can be reused in many different  places.  
Just because something can be inlined doesnt make it a versioned child.  Typically a versioned child has no independent  existance, thus, if its parent is deleted so is it.  Similalry the same  capacity requirement might be specified on many different vehicle types.

**Comment by ue71603** [comment]
@skinkie  Does Nick's anser satisfy you? Then we might close this issue.

**Comment by skinkie** [comment]
Not really :-)

**Comment by Aurige** [comment]
There are some missing substitution groups ... to be fixed

**Comment by ue71603** [comment]
I do this for 2.1. I guess this is hygiene only.
